scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -285,11 +315,17 @@ main (int argc, char *argv[])
   cmd.AddValue ("enableMobility", "Whether to enable mobility or simulate static scenario", enableMobility);
   cmd.AddValue ("verbose", "Turn on all WifiNetDevice log components", verbose);
   cmd.AddValue ("simulationTime", "Simulation time in seconds", simulationTime);
+  cmd.AddValue ("ac", "0: AC_BE, 1: AC_BK, 2: AC_VI, 3: AC_VO", ac);
   cmd.AddValue ("pcap", "Enable PCAP Tracing", pcapTracing);
   cmd.AddValue ("arrayConfig", "Antenna array configuration", arrayConfig);
+  cmd.AddValue ("seed", "Seed number for this simulation", seed);
+  cmd.AddValue ("run", "Run number for this simulation", run);
```

Comment:
Not needed.. Run and Seed numbers are already available in the command line, specifically with --RngRun and --RngSeed :)

Reply:
it's the same in the end

Reply:
I know, but if there's already something that does that for you, there's no need to add it

Reply:
I'd prefer the first syntax since then the line of code that launches the simulation is cleaner